internal/connectors/plugins/public/bankingcircle/plugin.go

```diff
@@ -6,19 +6,28 @@ import (
 	"github.com/formancehq/payments/internal/connectors/plugins"
 	"github.com/formancehq/payments/internal/connectors/plugins/public/bankingcircle/client"
 	"github.com/formancehq/payments/internal/models"
+	"github.com/hashicorp/go-hclog"
 )
 
 type Plugin struct {
+	logger hclog.Logger
+
 	client *client.Client
 }
 
+func New(logger hclog.Logger) models.Plugin {
```

Comment:
This is a little less idiomatic now because `bankingcircle.New()` returns `*bankingcircle.Plugin` rather than the expecte `*bankingcircle.BankingCircle`. Maybe stick to `bankingcircle.NewPlugin()` ?